### ISSUE TITLE
Add periodical visualization during training

### ISSUE THREAD

**Issue by toml01** [BioCellColman/BioCell#6]
Plot created images and label images to compare
_refs:_
- _https://keras.io/callbacks/_
- _https://www.tensorflow.org/tensorboard/image_summaries_

**Comment by toml01** [comment]
https://github.com/BioCellColman/BioCell/commit/951080e8cd8d3a2c027661fc4d6d2da317b1765d
